src/assets/GLSLAsset.js

```diff
@@ -0,0 +1,62 @@
+const Asset = require('../Asset');
+const localRequire = require('../utils/localRequire');
+const path = require('path');
+
+class GLSLAsset extends Asset {
+  constructor(name, pkg, options) {
+    super(name, pkg, options);
+    this.type = 'js';
+  }
+
+  async parse() {
+    await localRequire('glsl-token-inject-block', this.name);
```

Comment:
Why is this being required but not used?

Reply:
For some reason `yarn install` doesn't fetch this dependency (it's a dependency of `glsl-inject-defines`), but `npm install` does. As soon as we merge https://github.com/parcel-bundler/parcel/pull/805, it won't be needed. So this is a temporary workaround.

Reply:
@mvlabat Can you elaborate on why `yarn install` doesn’t work but `npm install` does. Not really sure if we should leave it broken like that, and not really sure if #805 is gonna solve anything anyways...

Reply:
@devongovett, @davidnagli, after rebasing with the latest master I couldn't reproduce the bug. So I've just removed that `localRequire`